Makefile

```diff
@@ -292,17 +292,26 @@ bundle-community: bundle ##Add Community Edition suffix to operator name
 	sed -r -i "s|displayName: Self Node Remediation Operator.*|displayName: Self Node Remediation Operator - Community Edition|;" ${BUNDLE_CSV}
 
 .PHONY: bundle-update
-bundle-update: ## Update containerImage, createdAt, skipRange, and icon fields in the bundle's CSV, then validate the bundle directory
+bundle-update: verify-previous-version ## Update containerImage, createdAt, skipRange, and icon fields in the bundle's CSV, then validate the bundle directory
```

Comment:
you might want to update the comment with the new `replaces` field, or just change it to a more generic "update CSV's fields and validate the bundle directory"